### DISCUSSION TITLE
Fixed size..

### DISCUSSION THREAD

**Discussion by danboyce92** [matteobruni/tsparticles#4893] [Q&A] [answered]
Hey there,
I have just disabled fullscreen but now have an issue with fitting properly inside the div.
no matter how I change the dimensions of the div my Particle component is in the rendered strip is much shorter than the div. How do I make the component fill the div it's in?
Is there an option I can edit or something?

**Comment by matteobruni** [comment, marked as answer]
If you disable the full screen option, it’s a standard html element, you have to set the css configuration as needed. There is the `style` property for changing canvas style, but if you target a div with the `tsParticles.load` function, the canvas will be inside that div with height and width at 100%

**Reply by JusticeMatthew** [reply]
How does this work in the react version? I don't use `tsParticles.load` so I'm not sure where to tell it what component to be inside of

**Reply by matteobruni** [reply]
The `Particle` component targets a div with a canvas. If you disable the `fullScreen` options, the id parameter can be used for customization

**Reply by JusticeMatthew** [reply]
That's what I've tried but despite disabling Fullscreen in the options the behavior never changes and it continues to ignore any css styling :(

**Reply by matteobruni** [reply]
Open an issue on [React repository](https://github.com/tsparticles/react) with your code

**Reply by deeNair** [reply]
having same issue
